lib/src/test/kotlin/integration/utils/IntegrationConfig.kt

```diff
@@ -65,6 +67,8 @@ class IntegrationConfig {
                 privateKey = Felt.fromHex(
                     env.getOrElse("INTEGRATION_PRIVATE_KEY") { throw RuntimeException("INTEGRATION_PRIVATE_KEY not found in environment variables") },
                 ),
+                constNonceAccountAddress = env["INTEGRATION_CONST_NONCE_ACCOUNT_ADDRESS"]?.let { Felt.fromHex(it) },
+                constNoncePrivateKey = env["INTEGRATION_CONST_NONCE_PRIVATE_KEY"]?.let { Felt.fromHex(it) },
```

Comment:
is there any info about this in readme/else?
Why not just reuse acountAddress/privateKey? you can just export different env file (that of const nonce account) and have the same functionality, right?

Reply:
> is there any info about this in readme/else?

I thought it might be better to leave for internal use only to not further confuse anyone, but I can add some basic info to readme.

>Why not just reuse acountAddress/privateKey? 

We are reusing acountAddress/privateKey it if constNonceAccountAddress/PrivateKey is not passed, but it could lead to unpredictable behaviour. I added the comment explaining that due to getNonce having access to pending block it retrieves higher nonce than other methods expect which leads to those methods failing.

> you can just export different env file (that of const nonce account) and have the same functionality, right?

Not really, because some tests change the state and some don't and the idea is to be able to run both simultaneously if needed: like fetching the eth balance and sending a transfer transaction. The need to supply a different env file each time would diminish the idea of having to use just one flag to switch between test mode and would be even harder to explain in readme. Not having info on what tests rely on constant nonce address would likely lead to someone running a increasing nonce test case, disrupting the essential function of using a constant nonce address, as the 'constant' nonce wouldn't remain constant anymore.